spec/models/base_organisation_spec.rb

```diff
@@ -56,5 +56,37 @@
       org.save
     end
   end
+  
+  describe 'friendly_id' do
+    # TODO: more tests on friendly_id methods?
```

Comment:
I didn't create too many tests. I just checked each "slug case" once. If we
delete some of them, some parts of the slug parser would stay untested.

2016-05-09 11:49 GMT+02:00 Sam Joseph <notifications@github.com>:

> In spec/models/base_organisation_spec.rb
> <https://github.com/AgileVentures/LocalSupport/pull/292#discussion_r62474064>
> :
>
> > @@ -56,5 +56,37 @@
> >        org.save
> >      end
> >    end
> > +
> > +  describe 'friendly_id' do
> > +    # TODO: more tests on friendly_id methods?
>
> do we need more tests? what does the coverage look like?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/AgileVentures/LocalSupport/pull/292/files/bf7b03ed1d65dedf261c812acd715258d6344307#r62474064>
>